packages/auth/src/graphql/resolvers/grant.test.ts

```diff
@@ -484,7 +484,7 @@ describe('Grant Resolvers', (): void => {
       const gqlQuery = appContainer.apolloClient
         .mutate({
           mutation: gql`
-            query GetGrant($id: String!) {
+            query GetGrant($id: ID!) {
```

Comment:
This test was passing because it was throwing an `ApolloError`, but not for the reason this test was looking for - it was erroring over a `String!` being passed into a parameter that expected an `ID!` type instead.